docs/configuration/install.md

```diff
@@ -249,31 +214,34 @@ On RHEL/CentOS:
 systemctl restart httpd.service
 ```
 
-<a name="php-installationconfiguration"></a>
-## PHP Installation/Configuration
+<a name="php-installation"></a>
+### PHP Installation and Configuration
 
-On Debian/Ubuntu 14.04 or 16.04 execute:
+On Debian/Ubuntu execute:
 ```
-apt-get install libapache2-mod-php7.1 php7.1-gd php7.1-mysql php7.1-mbstring php7.1-curl php7.1-dom php7.1-zip php-xml
+apt-get install libapache2-mod-php7.3 php7.3-gd php7.3-mysql php7.3-mbstring php7.3-curl php7.3-dom php7.3-zip php7.3-xml
```

Comment:
I would recommend to get rid of this. It is hard to keep track of and changes can be sudden. Additionally, if you need this instructions and follow them by the letter, you may end up with a system including eol packages really quickly.

My suggestion:
```suggestion
Please refere to the documentation of your installation to install either php7.3 or php7.4 including the gd, mysql, mbstring, curl, dom, zip, and xml modules. 
```